derp/derp.go

```diff
@@ -85,7 +85,7 @@ const (
 
 	// framePeerPresent is like framePeerGone, but for other
 	// members of the DERP region when they're meshed up together.
-	framePeerPresent = frameType(0x09) // 32B pub key of peer that's connected + optional 18B ip:port (16 byte IP + 2 byte BE uint16 port)
+	framePeerPresent = frameType(0x09) // 32B pub key of peer that's connected + optional 18B ip:port (16 byte IP + 2 byte BE uint16 port) + optional 1 byte peerPresentFlags
```

Comment:
Maybe say the frame size is always 41?

Reply:
Well, it's not necessarily, so that's overpromising. Old servers send less. The protocol is designed to be gracefully extended by making message types larger. You document what prefixes you might expect. I'll adjust the notation/comment on the double optional one.